### PR TITLE
scripts: add software Trusted Platform Module to CIV

### PR DESCRIPTION
**Pull request by GangSecurity**
1. install libtpms and swtpm library
2. start swtpm service when using start_flash_usb.sh
3. start swtpm service when launching android

Tracked-On: OAM-91814
Signed-off-by: Chen Gang G <gang.g.chen@intel.com>

### CODE REVIEW

File: scripts/setup_host.sh [hunk 1]
```diff
@@ -201,6 +201,37 @@ function setup_sof() {
...
+TPMS_LIB=libtpms-0.7.3
+SWTPM_VER=v0.3.3
+SWTPM=swtpm-0.3.3
+function ubu_install_swtpm() {
+        #install libtpms
+        apt-get -y install automake autoconf gawk
+        [ ! -f $TPMS_VER.tar.gz ] && wget https://github.com/stefanberger/libtpms/archive/$TPMS_VER.tar.gz -P $CIV_WORK_DIR
+        [ -d $CIV_WORK_DIR/$TPMS_LIB ] && rm -rf  $CIV_WORK_DIR/$TPMS_LIB
+        tar zxvf $CIV_WORK_DIR/$TPMS_VER.tar.gz
+        tar zxvf $CIV_WORK_DIR/$TPMS_VER.tar.gz
```
Comment (James-Zzz) [inline scripts/setup_host.sh:214]: typo? Unzip TPMS_VER.tar.gz twice?

File: scripts/setup_host.sh [hunk 2]
```diff
@@ -201,6 +201,37 @@ function setup_sof() {
...
+        cd $CIV_WORK_DIR/$TPMS_LIB
+        tar zxvf $CIV_WORK_DIR/$TPMS_VER.tar.gz
```
Comment (James-Zzz) [inline scripts/setup_host.sh:216]: typo? Unzip TPMS_VER.tar.gz twice?

File: scripts/setup_host.sh [hunk 3]
```diff
@@ -201,6 +201,35 @@ function setup_sof() {
     $CIV_WORK_DIR/scripts/setup_audio_host.sh
 }
 
+TPMS_VER=v0.7.3
+TPMS_LIB=libtpms-0.7.3
+SWTPM_VER=v0.3.3
+SWTPM=swtpm-0.3.3
```
Comment (YadongQi) [inline scripts/setup_host.sh:207]: make these variables local variables in function ubu_install_swtpm().

File: scripts/start_civ.sh [hunk 4]
```diff
@@ -397,6 +404,7 @@ function launch_guest() {
                    $GUEST_TIME_KEEP \
                    $GUEST_STATIC_OPTION \
                    $GUEST_EXTRA_QCMD \
+                   $GUSET_VTPM \
```
Comment (YadongQi) [inline scripts/start_civ.sh:407]: move this to ahead of $GUEST_STATIC_OPTION

File: scripts/start_flash_usb.sh [hunk 5]
```diff
@@ -38,6 +38,11 @@ else
 	display_type="gtk,gl=on"
 fi
 
+#start software Trusted Platform Module
+WORK_DIR=$PWD
```
Comment (YadongQi) [inline scripts/start_flash_usb.sh:42]: move this definition to the beginning of the script.

**Comment by YadongQi** [comment]
Please change the patch title label from "vm_manager:" to "scripts:"

File: scripts/setup_host.sh [hunk 6]
```diff
@@ -201,6 +201,35 @@ function setup_sof() {
...
+function ubu_install_swtpm() {
+        #install libtpms
```
Comment (YadongQi) [inline scripts/setup_host.sh:209]: replace 8 spaces with 4 space indent.

**Pull request merged**